phylogenetic/rules/prepare_sequences.smk

```diff
@@ -94,8 +94,8 @@ rule filter:
             --sequences {input.sequences:q} \
             --metadata {input.metadata:q} \
             --metadata-id-columns {params.id_field:q} \
-            --query "{params.query}" \
             {params.filter_params} \
+            --query "({params.min_coverage} & ({params.genogroup_query}))" \
```

Comment:
As an outside user, I'd be very confused by the interaction between the `filter_params` and the built in query in the workflow. The internal query uses `is_lab_host` and seems to be dependent on the `filter_params` declaring `--query-columns is_lab_host:str`, which is not apparent just from looking at the config.yaml. Also note, that the built-in `--query` would override any user provided `--query` option in `filter_params`. 

I suggest stepping back a bit here to reconsider the config params around this filter rule.